bevy_rapier_benches3d/README.md

```diff
@@ -10,6 +10,13 @@ and outputs them at the end.
 cargo run --release --bin bench
```

Comment:
Running `cargo run --release --bin bench` from the root brings all dependencies, but running this from `bevy_rapier_benches3d` brings only the needed ones, this may be worth a note (and I'd like to understand why.)